keps/sig-release/2572-release-cadence/kep.yaml

```diff
@@ -0,0 +1,37 @@
+title: Defining the Kubernetes Release Cadence
+kep-number: 2572
+authors:
+  - "@justaugustus"
+owning-sig: sig-release
+participating-sigs:
+  - sig-architecture
+  - sig-testing
+status: implementable
+creation-date: 2021-01-21
+reviewers:
+  - "@BenTheElder"
+  - "@derekwaynecarr"
+  - "@dims"
+  - "@ehashman"
+  - "@hasheddan"
+  - "@jeremyrickard"
+  - "@johnbelamaric"
+  - "@spiffxp"
+  - "@stevekuznetsov"
+approvers:
+  - "@LappleApple"
+  - "@saschagrunert"
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.22"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.22"
+  beta: "v1.23"
+  stable: "v1.25"
```

Comment:
Hey Josh, my understanding around this point and the 1.25 date was essentially this:

* The change starts, in a probationary manner, with 1.22.
* We collect feedback and after three minor version (1.22, 1.23, 1.24) are complete we would work to finalize this to "stable" in 1.25.

Reply:
So that wasn't at all clear to many people, including several of those who commented on the KEP.  As a result, you cannot assume that a +1 on the KEP is actually a +1 on changing the schedule in 2021, since many people (including me) +1'd it thinking that GA was when the schedule would actually change.  This means that you need to check support for the timing now as a separate issue.

Reply:
And hey, I wanna say sorry for putting this on you, I know you didn't pick the timeline for either the proposed change or the date the KEP was filed.